public/app/core/utils/fetch.ts

```diff
@@ -103,31 +103,31 @@ export async function parseResponseBody<T>(
   if (responseType) {
     switch (responseType) {
       case 'arraybuffer':
-        return response.arrayBuffer() as any;
+        return response.arrayBuffer() as Promise<T>;
 
       case 'blob':
-        return response.blob() as any;
+        return response.blob() as Promise<T>;
```

Comment:
Unsure how I feel about these - these two methods return an explicit type (`Promise<ArrayBuffer>` and `Promise<Blob>`), but this `parseResponseBody` method has a return type of `Promise<T>` so it was always returning this type anyway.... 

I guess it still has the `as` assertions, so it'll still get flagged as something to fix?

I think this might need a bigger refactor later. Maybe just add a TODO comment saying yeah, these return specific types?